### PR TITLE
lesson 9 Inheritance & some Netflix

### PR DESCRIPTION
**Pull request by alinko-sharapova**

### CODE REVIEW

File: docs/lessons/09-interitance.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# 09 - Inheritance & overriding
+<Teacher name="Alina"></Teacher>
+
+## Goals
+- Understand what is Inheritance and why do we need it 
+- Understand what is Overriding
+- Understand what is difference between Overriding and Overloading
```
Comment (ftchirou) [inline docs/lessons/09-interitance.md]: I think `the` is missing between `is` and `difference` here.

File: docs/lessons/09-interitance.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+1.1. `Student` has a list of subjects that they study
+1.2. `Teacher` has a subject that they teach, how many years of experience they have
+1.3. Add a method `study(String subject)` to the `Student` class, that prints
+1.4. Add a method `teach()` to the `Teacher` class
+2. Let's add constructors to the `Person`, `Student` & `Teacher` classes
+3. Let's override `introduce()` method of the `Student` and `Teacher` classes
+
+### What you can do in a Subclass
+
+A subclass inherits all of the public and protected members of its parent, no matter what package the subclass is in. If the subclass is in the same package as its parent, it also inherits the package-private members of the parent. You can use the inherited members as is, replace them, hide them, or supplement them with new members:
```
Comment (ftchirou) [inline docs/lessons/09-interitance.md:71]: Have we, in a previous class, introduced the notion of package to the students? Or will it be done in this class?

**Review by ftchirou — APPROVED** [review]

File: docs/lessons/09-interitance.md (hunk 1 above)
Comment (alinko-sharapova) [reply docs/lessons/09-interitance.md]: Hate articles ...

File: docs/lessons/09-interitance.md (hunk 2 above)
Comment (alinko-sharapova) [reply docs/lessons/09-interitance.md:71]: good point, I will do it right before Access modifiers recap

**Pull request merged**